app/controllers/calendars_controller.rb

```diff
@@ -0,0 +1,75 @@
+class CalendarsController < ApplicationController
+  before_action :set_calendar, only: [:show, :edit, :update, :destroy]
+
+  # GET /calendars
+  # GET /calendars.json
+  def index
+    @calendars = Calendar.all
+  end
+
+  # GET /calendars/1
+  # GET /calendars/1.json
+  def show
+    @events = @calendar.events
```

Comment:
I added this (I can do this because of the has_many and belongs_to relationship between calendars and events). This gives me all events belonging to the calendar whose show page we are currently on.